polars/polars-lazy/polars-plan/src/dsl/dt.rs

```diff
@@ -180,6 +214,24 @@ impl DateLikeNameSpace {
             .map_private(FunctionExpr::TemporalExpr(TemporalFunction::TimeStamp(tu)))
     }
 
+    #[cfg(feature = "truncate")]
```

Comment:
Out of interested what is the relevance of the map_private implementation? It seems to work without.